Language reworked (#30)
iNPUTmice/lttrs-android: README.md

# Ltt.rs for Android

Proof of concept e-mail (JMAP) client (pronounced \"Letters\").
Makes heavy use of Android Jetpack to be more maintainable than some of the other Android e-mail clients.

![Android screenshot](https://gultsch.de/files/lttrs-android.png)

If the above screenshots don’t do enough to convince you, you can watch this
[short video from invidio.us](https://www.invidio.us/watch?v=ArCuudFwJX4).

> ### Feature and design considerations:

Author: ```suggestion
### Features, and design considerations:
```

> * _Heavily cached_ but not fully offline capable. Ltt.rs makes use of JMAP’s great caching capabilities. Marking a thread as read however need to a round-trip to the server for things like unread count to be updated. The action itself won’t get lost even if performed offline.

Reviewer: need to a round-trip?

Author: Good catch. I think I meant to write "to do a", but I went with something shorter now that I thought about it.

> * _Heavily cached_ but not fully offline capable. Ltt.rs makes use of JMAP’s great caching capabilities. Marking a thread as read however need to a round-trip to the server for things like unread count to be updated. The action itself won’t get lost even if performed offline.

Author: ```suggestion
* _Heavily cached_, but not fully offline capable. Ltt.rs makes use of JMAP’s great caching capabilities. However, marking a thread as read does round-trip to the server to be update things such as read count. The action itself won’t get lost even if performed offline.
```

* Account _setup and done_. Settings invite feature creep and its friend unmaintainability. There is one specific work flow. K-9 Mail or FairEmail is more suitable for others.
* _Minimal dependencies_. Third party libraries are often of poor quality, and end up unmaintained. Only widely known, highly tested libraries from reputable vendors.
* _First class Autocrypt_¹. Ltt.rs fits right into its strict UX guidelines.

> * _Based on jmap-mua_, a headless e-mail client, or a library that handles everything an e-mail client would, aside from data storage and UI. There is also lttrs-cli, which uses the same library.

Author: "E-mail" produces fewer errors in translation because variants without the dash aren't allowed in most other languages.
